src/PayPal/Order/DTO/PayPalOrderDTO.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShop\Module\PrestashopCheckout\PayPal\Order\DTO;
+
+class PayPalOrderDTO
+{
+
+    /**
+     * @var int
+     */
+    private $id;
+    /**
+     * @var int
+     */
+    private $idCart;
+    /**
+     * @var string
+     */
+    private $intent;
+    /**
+     * @var string
+     */
+    private $fundingSource;
+    /**
+     * @var string
+     */
+    private $status;
+    /**
+     * @var array
+     */
+    private $paymentSource;
+    /**
+     * @var string
+     */
+    private $environment;
```

Comment:
Is it the paypal mode ? SANDBOX or LIVE ? If it's the case, it's better call it `$mode`